Library/Formula/dockviz.rb

```diff
@@ -0,0 +1,19 @@
+class Dockviz < Formula
+  desc "Visualizing docker data"
+  homepage "https://github.com/justone/dockviz"
+  url "https://github.com/justone/dockviz.git", :tag => "v0.3", :revision => "ae855d6a5a540c3563948e7035f9d19cab17f5b0"
```

Comment:
Please leave `tag` and `revision` on their own lines. Helps readability a bit, Thanks!